### PR TITLE
Refactored the storage of original and modified images

### PR DESCRIPTION
**Pull request by FawcettJohnW**

### CODE REVIEW

File: image_scraper/scraper.py [hunk 1]
```diff
@@ -1,73 +1,102 @@
...
+            print(f"there was an issue saving {filename}")
+            print(f"the source url is: {url}")
+            print(e)
+            return
+        print(f"Image saved as {filename}")
+        
+def save_original_image(originalCounter, url):
+    dirname = f"processing_queue/original"
+    isExist = os.path.exists(dirname)
+    if not isExist:
```
Comment (zfolwick) [inline image_scraper/scraper.py:71]: this is vastly more pleasant than what I was doing.

File: image_scraper/scraper.py [hunk 2]
```diff
@@ -1,73 +1,102 @@
...
+        
+def save_images(classification, originalCounter, urls):
+    for url in urls:
+        save_image(classification, originalCounter, url)
+#%%
+# just a utility
+import secrets
+import string
+
+def generate_random_string(length):
```
Comment (zfolwick) [inline image_scraper/scraper.py:98]: I see no reason to keep this.  I'll mark it for deletion later

File: image_scraper/scraper.py [hunk 3]
```diff
@@ -1,73 +1,102 @@
...
+	shopped_image_assets = []
+	for i in shopped_images:
+		current = str(i)
+		if ('imgur' in current and not 'desktop-assets' in current and not 'favicon' in current):
+			shopped_image_assets.append(i['src'].split('?')[0])
+	return list(set(shopped_image_assets))
+
+#%%
+# Files in windows OS cannot have "?", "&" or superfluous ".", and "/" just messes things up.
+def formatForFs(url):
```
Comment (zfolwick) [inline image_scraper/scraper.py:36]: I'm not convinced we should care about this, I just wanted a way to move backwards to go to the url if needed.  I can see just removing this in favored of numbered files.

**Review by zfolwick — APPROVED** [review]
👍 LGTM

File: image_scraper/scraper.py (hunk 3 above)
Comment (FawcettJohnW) [reply image_scraper/scraper.py:36]: I think that's cool.  These file names are pretty long

**Pull request merged**